core/module/digital_twin_registry/src/digital_twin_registry_module.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+use common::grpc_module::GrpcModule;
+use core_protobuf_data_access::module::digital_twin_registry::v1::digital_twin_registry_server::DigitalTwinRegistryServer;
+
+use tonic::transport::server::RoutesBuilder;
+
+use crate::digital_twin_registry_impl::DigitalTwinRegistryImpl;
+
+/// Digital Twin Registry Module.
+#[derive(Clone, Debug)]
+pub struct DigitalTwinRegistryModule {}
+
+impl DigitalTwinRegistryModule {
+    /// Creates a new instance of the DigitalTwinRegistryModule.
+    pub async fn new() -> Result<Self, tonic::Status> {
```

Comment:
nit: this constructor is unnecessary, clients can just use `DigitalTwinRegistryModule { }`

Reply:
Leaving this in for consistency with other struct's implementation.